lib/components/widgets/tweet/tweet_tile_content.dart

```diff
@@ -380,15 +381,34 @@ class TweetActionsRow extends StatelessWidget {
           favorite: model.favorite,
           unfavorite: model.unfavorite,
         ),
+        Spacer(),
         if (model.allowTranslation) ...[
-          Spacer(),
           _TweetTranslationButton(model),
         ],
+        TweetShareButton(model.tweet),
```

Comment:
I think it would be nice to have a popup menu for actions such as sharing and deleting a Tweet. Maybe even an action to hide someones tweet.
We can keep it like this and change it later.

Reply:
Yes. Maybe we should do that with https://github.com/robertodoering/harpy/issues/143.